docs/components/table-view.md

```diff
@@ -0,0 +1,1016 @@
+# Table View
+
+Component for displaying data in table format.
+
+:::tip NOTE
+KTableView is a display-only component, it does not provide any data management capabilities like fetching, pagination, sorting or searching. If you are looking for a component that includes those functions out of the box, check out [KTable](/components/table).
+:::
+
+<KTableView
+  :data="basicData"
+  :headers="basicHeaders()"
+/>
+
+```html
+<KTableView
+  :data="tableData"
+  :headers="headers"
+/>
+```
+
+## Props
+
+### headers
+
+Array of objects that represent table columns along with some configurations that should apply to each of the columns (whether a column is sortable, hidable, etc).
+
+```ts
+interface TableViewHeader {
+  key: string | 'actions' // must be unique for each column, see Reserved Header Keys section for more information about 'actions' key value
+  label: string // visible column header text
+  sortable?: boolean // in a nutshell, this property defines whether sort icon should be displayed next to the column header and whether the column header will emit sort event upon clicking on it
+  hidable?: boolean // allow toggling column visibility
+  tooltip?: string // when provided, an info icon will be rendered next to the column label, upon hovering on which the tooltip will be revealed
+  hideLabel?: boolean // whether column header text should be hidden (only visible to screen readers)
+}
+```
+
+:::tip NOTE
+If at least one column is `hidable` in the table, KTableView will render a dropdown on the right of the table toolbar, directly above the table, which will provide an interface for showing/hiding columns to the user.
+:::
+
+For an example of `headers` prop usage please refer to [`data` prop documentation](#data) below.
+
+#### Reserved Header Keys
+
+:::warning NOTE
+This feature is only available in KTableView and not available in [KTable](/components/table).
+:::
```

Comment:
```suggestion
```
I don't think we need these notes; there is no assumption that props are available in both components or that the component interfaces are the same.